include/network.php

```diff
@@ -126,6 +126,7 @@ function z_fetch_url($url, $binary = false, &$redirects = 0, $opts = array())
 	if (x($opts, 'timeout')) {
 		@curl_setopt($ch, CURLOPT_TIMEOUT, $opts['timeout']);
 	} else {
+		// if Config::get returns null, intval will be 0 which will also evaluate to false
```

Comment:
This is wrong, no matter what `intval($var) !== false` will be true, even if `$var = false`.


Here's how it should be:
```php
$curl_time = Config::get('system', 'curl_timeout', 60);
@curl_setopt($ch, CURLOPT_TIMEOUT, intval($curl_time));
```

Reply:
Yeah, I guess it is better to change the config call so that we provide a default value.